src/CMakeLists.txt

```diff
@@ -137,7 +137,11 @@ add_library(parthenon
   parameter_input.cpp
   parthenon_manager.cpp
 )
+
 target_compile_features(parthenon PUBLIC cxx_std_14)
+if (CMAKE_CXX_COMPILER_ID STREQUAL "XL")
+  target_compile_options(parthenon PUBLIC -std=c++1y -qxflag=disable__cplusplusOverride)
```

Comment:
Why is "qxflag=disable__cplusplustOverride" needed, I'm having trouble finding what it is used for?

Reply:
See the note at the bottom of this page: https://www.ibm.com/support/knowledgecenter/SSXVZZ_16.1.1/com.ibm.xlcpp1611.lelinux.doc/language_ref/standard_features.html